openeo_udp/biopar/biopar.json

```diff
@@ -0,0 +1,174 @@
+            "rel": "canonical",
+            "title": "Public URL for user-defined process 'BIOPAR'"
+        },
+        {
+            "href": "https://marketplace-portal.dataspace.copernicus.eu/catalogue/app-details/21",
+            "rel": "about",
+            "title": "openEO Algorithm Plaza"
+        }
+    ],
+    "parameters": [
+        {
+            "description": "Left-closed temporal interval, i.e. an array with exactly two elements:\n\n1. The first element is the start of the temporal interval. The specified instance in time is **included** in the interval.\n2. The second element is the end of the temporal interval. The specified instance in time is **excluded** from the interval.\n\nThe specified temporal strings follow [RFC 3339](https://www.rfc-editor.org/rfc/rfc3339.html). Also supports open intervals by setting one of the boundaries to `null`, but never both.",
+            "name": "temporal_extent",
+            "schema": {
+                "examples": [
+                    [
+                        "2015-01-01T00:00:00Z",
+                        "2016-01-01T00:00:00Z"
+                    ],
+                    [
+                        "2015-01-01",
+                        "2016-01-01"
+                    ]
+                ],
+                "items": {
+                    "anyOf": [
+                        {
+                            "format": "date-time",
+                            "subtype": "date-time",
+                            "type": "string"
+                        },
+                        {
+                            "format": "date",
+                            "subtype": "date",
+                            "type": "string"
+                        },
+                        {
+                            "maxLength": 4,
+                            "minLength": 4,
+                            "pattern": "^\\d{4}$",
+                            "subtype": "year",
+                            "type": "string"
+                        },
+                        {
+                            "type": "null"
+                        }
+                    ]
+                },
+                "maxItems": 2,
+                "minItems": 2,
+                "subtype": "temporal-interval",
+                "type": "array"
+            }
+        },
+        {
+            "description": "This geometries defines the bounds of the resulting image. Pixels outside the polygon are set to nodata.",
+            "name": "geometries",
+            "schema": {
+                "subtype": "geojson",
+                "type": "object"
+            }
+        },
+        {
+            "default": "FAPAR",
+            "description": "BIOPAR type [FAPAR,LAI,FCOVER,CCC,CWC]",
+            "name": "biopar_type",
+            "optional": true,
+            "schema": {
+                "type": "string"
+            }
```

Comment:
This parameter seems to be a enum, with predefined possible values. If that is the case, the schema should reflect that properly.
For example see https://github.com/Open-EO/openeo-processes/blob/6141771129d58d2a292db3b91d80c812c49f4e52/add_dimension.json#L37-L50  for an example


Also note that if you use the python client to build the UDP, you can do that with the `values` argument of `Parameter.string` (https://open-eo.github.io/openeo-python-client/api.html#openeo.api.process.Parameter.string)